### ISSUE TITLE
undefined method for nil:NilClass

### ISSUE THREAD

**Issue by radist** [RealEnder/redmine_auditlog#1]
```
$ rails generate audited:install --audited-changes-column-type jsonb RAILS_ENV=production
/home/redmine/web/rm.rubium.ru/public_html/plugins/redmine_sidekiq/lib/redmine_sidekiq/configure.rb:8:in `<class:Configure>': undefined method `[]' for nil:NilClass (NoMethodError)
	from /home/redmine/web/rm.rubium.ru/public_html/plugins/redmine_sidekiq/lib/redmine_sidekiq/configure.rb:4:in `<module:RedmineSidekiq>'
	from /home/redmine/web/rm.rubium.ru/public_html/plugins/redmine_sidekiq/lib/redmine_sidekiq/configure.rb:3:in `<top (required)>'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/redmine/web/rm.rubium.ru/public_html/plugins/redmine_sidekiq/init.rb:1:in `<top (required)>'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/redmine/web/rm.rubium.ru/public_html/lib/redmine/plugin.rb:155:in `block in load'
	from /home/redmine/web/rm.rubium.ru/public_html/lib/redmine/plugin.rb:146:in `each'
	from /home/redmine/web/rm.rubium.ru/public_html/lib/redmine/plugin.rb:146:in `load'
	from /home/redmine/web/rm.rubium.ru/public_html/config/initializers/30-redmine.rb:21:in `<top (required)>'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:268:in `load'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:268:in `block in load'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:268:in `load'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/engine.rb:652:in `block in load_config_initializer'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/notifications.rb:166:in `instrument'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/engine.rb:651:in `load_config_initializer'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/engine.rb:616:in `block (2 levels) in <class:Engine>'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/engine.rb:615:in `each'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/engine.rb:615:in `block in <class:Engine>'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/initializable.rb:30:in `instance_exec'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/initializable.rb:30:in `run'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/initializable.rb:55:in `block in run_initializers'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:228:in `block in tsort_each'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:431:in `each_strongly_connected_component_from'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/initializable.rb:44:in `each'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/initializable.rb:44:in `tsort_each_child'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:415:in `call'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:415:in `each_strongly_connected_component_from'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:349:in `block in each_strongly_connected_component'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:347:in `each'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:347:in `call'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:347:in `each_strongly_connected_component'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:226:in `tsort_each'
	from /home/redmine/.rvm/rubies/ruby-2.3.3/lib/ruby/2.3.0/tsort.rb:205:in `tsort_each'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/initializable.rb:54:in `run_initializers'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/application.rb:352:in `initialize!'
	from /home/redmine/web/rm.rubium.ru/public_html/config/environment.rb:14:in `<top (required)>'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/application.rb:328:in `require_environment!'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:142:in `require_application_and_environment!'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:128:in `generate_or_destroy'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /home/redmine/.rvm/gems/ruby-2.3.3/gems/railties-4.2.7.1/lib/rails/commands.rb:17:in `<top (required)>'
	from bin/rails:4:in `require'
	from bin/rails:4:in `<main>'
```

**Comment by RealEnder** [comment]
This breaks in redmine_sidekiq plugin, when installing. Can you try to remove redmine_sidekiq and reinstall gem to confirm the issue there?

**Comment by RealEnder** [comment]
Closed as per #7
